Update README.md

NiranjanNlc opened the pull request:
Adding Sherpa tech information

NiranjanNlc committed:
Update README.md

techgaun reviewed (changes requested):
The URL does not seem like working, sounds like `www` is the culprit there. https://sherpa-tech.com/ seems to work though

NiranjanNlc committed:
Updated according to suggested change 

Actually, www. is not culprit , the - in domain name if goes at the end of line didnot take tech.com also as the url . 
However, removing www. should work any

NiranjanNlc reviewed (commented):
Updated by removing www

techgaun commented on README.md:
```diff
@@ -79,7 +79,7 @@ If you cannot provide a valid website for your company your PR will not be accep
 ## S - U
 - [SPI Nepal](http://www.softvision.com/) | Thapathali, Kathmandu | Provider of Digital Solutions, Mobility, Analytics, Merchandise Planning, In-Store Engagements and Quality Assurance
-- [Sherpa Tech](https://www.sherpa-tech.com/) |Jamal , Kathmandu |Software, Web development and Consulting company  since 2010.
+- [Sherpa Tech](https://sherpa-tech.com/) |Jamal , Kathmandu |Software, Web development and Consulting company  since 2010.
```
```suggestion
- [Sherpa Tech](https://sherpa-tech.com/) | Jamal, Kathmandu | Software, Web development and Consulting company since 2010.
```

NiranjanNlc committed:
Update README.md act suggestions

Co-authored-by: Samar Dhwoj Acharya <coolsamar207@gmail.com>
